lstchain/reco/r0_to_dl1.py

```diff
@@ -548,8 +548,8 @@ def r0_to_dl1(
     if is_simu:
         # Reconstruct source position from disp for all events and write the result in the output file
         for tel_name in ['LST_LSTCam']:
```

Comment:
Why a loop over a single entry?

Reply:
this is still related to the inability to work with more than 1 telescope (and it should be LST_LSTCam). I address it in #646